### PR TITLE
Improve agent details page

### PR DESCRIPTION
**Pull request by guidomodarelli**
### Description

We want to refine some aspects of the Agent dashboard to enhance the use experience. To achieve this we will do the following tasks.

### Issues Resolved

Closes https://github.com/wazuh/wazuh-dashboard-plugins/issues/7193

### Evidence

- [x] Adjusted the height of the event count evolution visualization.

  image

- [x] Standardized padding for the "Top 5 Packages" table to align with other tables.

  image

  image

- [x] Reduced the width of small fields in the Agent Info ribbon for a more compact layout.

  image

  image

- [X] Replaced "Malware Detection" with "Vulnerability Detection" in the agent's pinned navigation menu.

  image

- [x] Aligned "Vulnerability Detection" KPIs to the left.

  image

- [x] Max quantity of groups is 3

  image

### Test

1. Navigate to `Agents management` » `Summary`
2. Select any agent.
3. Review the checklist located in the Evidence section.

### Check List

- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [x] Update [CHANGELOG.md](./../CHANGELOG.md)
- [x] Commits are signed per the DCO using --signoff

### CODE REVIEW

**Review by Tostti — CHANGES_REQUESTED** [review]
# Test 🔴 
# CR 🟢 


| Test | Firefox | Chrome| Safari| 
|---|---|---|---|
| Adjusted the height of the event count evolution visualization.|🟢 |🟢|:black_circle:|
|Standardized padding for the "Top 5 Packages" table to align with other tables.|🟢|🟢|:black_circle:|
| Reduced the width of small fields in the Agent Info ribbon for a more compact layout.|🔴 |🔴 |:black_circle:|
|Replaced "Malware Detection" with "Vulnerability Detection" in the agent's pinned navigation menu.|🟢 |🟢|:black_circle:|
|Aligned "Vulnerability Detection" KPIs to the left.|🟢 |🟢|:black_circle:|


###  Adjusted the height of the event count evolution visualization. 🟢 

<details>

**Firefox**
imagen

**Chrome**
imagen

</details>


###  Standardized padding for the "Top 5 Packages" table to align with other tables. 🟢 

<details>

**Firefox**
imagen

**Chrome**
imagen

</details>

###  Reduced the width of small fields in the Agent Info ribbon for a more compact layout. 🔴 

<details>

On IPv6, the address superposes with `Version`

**Firefox**

imagen

**Chrome**
imagen

</details>

###  Replaced "Malware Detection" with "Vulnerability Detection" in the agent's pinned navigation menu. 🟢 

<details>

**Firefox**
imagen


**Chrome**
imagen

</details>

###  Aligned "Vulnerability Detection" KPIs to the left. 🟢 

<details>

**Firefox**
imagen

**Chrome**
imagen

</details>

**Comment by gdiazlo** [comment]
The padding / left margin on the left aligned vulnerability count panel looks smaller than it should be

**Review by Tostti — DISMISSED** [review]
Issue reported in https://github.com/wazuh/wazuh-dashboard-plugins/pull/7196#pullrequestreview-2485266941solved. 

## Test 🟢 
imagen

## CR 🟢 

LGTM

**Review by chantal-kelm — CHANGES_REQUESTED** [review]
🔴 Reduced the width of small fields in the Agent Info ribbon for a more compact layout.

Captura de pantalla 2024-12-10 a la(s) 3 21 56 p  m

🔴 Replaced "Malware Detection" with "Vulnerability Detection" in the agent's pinned navigation menu.

Captura de pantalla 2024-12-10 a la(s) 3 17 19 p  m

**Review by chantal-kelm — DISMISSED** [review]
CR: ✅ 
TEST: ✅ 

Captura de pantalla 2024-12-10 a la(s) 4 46 04 p  m

File: plugins/main/public/components/common/ribbon/ribbon-item.tsx [hunk 1]
```diff
@@ -51,9 +54,7 @@ interface RibbonItemProps {
 const WzRibbonItem = (props: RibbonItemProps) => {
   const { item } = props;
 
-  const elementStyle = { ...(item.style || {}), fontSize: FONT_SIZE };
-  const wzWidth100 = { anchorClassName: 'wz-width-100' };
-  const tooltipProps = item.key === 'cluster-node' ? wzWidth100 : {};
+  item.style = { ...item.style, fontSize: FONT_SIZE };
```
Comment (Desvelao) [inline plugins/main/public/components/common/ribbon/ribbon-item.tsx:57]: **nitpick**: this seems to redefine a prop and this is a bad practice. Create a new object/variable to append the `fontSize` definition.

File: plugins/main/public/components/common/ribbon/ribbon-item.tsx [hunk 2]
```diff
@@ -91,51 +92,53 @@ const WzRibbonItem = (props: RibbonItemProps) => {
     return `${name} ${version}`;
   };
 
-  const renderField = function <T>(field?: T): T | string {
+  const renderOptionalField = function <T>(field?: T): T | string {
     return field !== undefined || field ? field : '-';
   };
 
+  if (isOperatingSystem(item)) {
+    item.render = () => {
```
Comment (Desvelao) [inline plugins/main/public/components/common/ribbon/ribbon-item.tsx:100]: **nitpick**: this seems to redefine a prop and this is a bad practice. Create a new object/variable to append the `render` definition.

**Review by chantal-kelm — DISMISSED** [review]
CR: ✅ 
TEST: ✅ 

Captura de pantalla 2024-12-11 a la(s) 2 49 55 p  m

**Review by asteriscos — DISMISSED** [review]
CR: :green_circle: 
Test: :green_circle: 

#### Firefox
image


#### Chrome

image

**Review by asteriscos — DISMISSED** [review]

**Review by chantal-kelm — DISMISSED** [review]
Captura de pantalla 2024-12-11 a la(s) 3 09 37 p  m

**Review by chantal-kelm — APPROVED** [review]

**Review by asteriscos — APPROVED** [review]

**Pull request merged**